src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml

```diff
@@ -115,6 +116,15 @@
                            Foreground="#3C3C3C"
                            Text="{Binding BodyText, UpdateSourceTrigger=PropertyChanged}"
                            TextWrapping="Wrap" />
+                <localui:CustomRichTextBox x:Name="ContentRichTextBox" 
+                                           IsDocumentEnabled="True"
+                                           Visibility="Collapsed"
+                                           DockPanel.Dock="Left"
+                                           Grid.Row="1"
+                                           Margin="20"
+                                           CustomText="{Binding BodyText}"
+                                           Style="{StaticResource CustomRichTextBoxStyle}">
+                </localui:CustomRichTextBox>
```

Comment:
Would you also test if normal messages display correctly?

Reply:
Oh you are adding a RIchTextBox instead of forcing it, I guess it is less impactful then